### PR TITLE
bugfix: fixes bug where typing state was starting even on arrow key p…

### PR DESCRIPTION
**Pull request by ryanholte**
Fixes a bug where typing state would show even on arrow key press

### CODE REVIEW

**Comment by ryanholte** [comment]
@makepanic Any thoughts on this pull request?

**Comment by ryanholte** [comment]
@makepanic just wanted to make sure you've seen this.

**Comment by makepanic** [comment]
Hi sorry for the slow response, I'll review it soon.

**Comment by makepanic** [comment]
Thanks, this fix makes sense. I've released it as 1.0.1

File: addon/components/keyword-complete.js [hunk 1]
```diff
@@ -508,14 +521,6 @@ export default Ember.Component.extend({
...
-    if (visible && this.get('shouldShowTypingState')) {
-      this.set('showTypingState', true);
-      run.later(this, () => {
-        this.set('showTypingState', false);
-      }, this.get('typingStateTimeout'));
-    }
-
     let selectionIdx = this.get('selectionIdx');
     switch (keyCode) {
       case KEYS.ENTER:
```
Comment (Prince11112) [inline addon/components/keyword-complete.js:526]: #4

**Pull request merged**